torchvision/csrc/io/image/cuda/encode_decode_jpegs_cuda.h

```diff
@@ -11,5 +12,9 @@ C10_EXPORT torch::Tensor decode_jpeg_cuda(
     ImageReadMode mode,
     torch::Device device);
 
+C10_EXPORT std::vector<torch::Tensor> encode_jpegs_cuda(
```

Comment:
Add a comment here or somewhere for the user to say that it only supports contiguous tensors?

Reply:
Line 87 in encode_jpegs_cuda.cpp should takes care of handling non-contiguous images.